gdbstub.h

```diff
@@ -0,0 +1,3 @@
+#pragma once
+
+extern struct target_ops rv_ops;
```

Comment:
Move the `extern` to `gdbstub.c` and delete file `gdbstub.h`.

Reply:
Then, put `extern` in file `emulate.c` as well. It is not worthy to maintain a header file containing one declaration.